File: app/src/main/java/it/cammino/risuscito/database/entities/ListaPers.java
package it.cammino.risuscito.database.entities;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import it.cammino.risuscito.ListaPersonalizzata;

@Entity
public class ListaPers {

    @PrimaryKey(autoGenerate = true)
    public int id;

    public String titolo;

    @ColumnInfo(typeAffinity = ColumnInfo.BLOB)
    public ListaPersonalizzata lista;
}

File: app/src/main/java/it/cammino/risuscito/database/entities/ListaPers.kt
package it.cammino.risuscito.database.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

import it.cammino.risuscito.ListaPersonalizzata

@Entity
class ListaPers {

    @PrimaryKey(autoGenerate = true)
    var id: Int = 0

    var titolo: String? = null

    @ColumnInfo(typeAffinity = ColumnInfo.BLOB)
    var lista: ListaPersonalizzata? = null
}
